airbyte-integrations/bases/standard-source-test/build.gradle

```diff
@@ -3,10 +3,6 @@ plugins {
     id 'airbyte-docker'
 }
 
-application {
```

Comment:
Why is this removed?

Reply:
it's duplicated -- see below in the file